### PR TITLE
daemontools: init at 0.76

### PR DESCRIPTION
**Pull request by kevincox**
###### Things done
- [x] Tested using sandboxing (`nix-build --option build-use-chroot true` or [nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [x] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/admin/daemontools/default.nix [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+stdenv.mkDerivation rec {
+  name = "daemontools-0.76";
+  
+  src = fetchurl {
+    url = "https://cr.yp.to/daemontools/${name}.tar.gz";
+    sha256 = "07scvw88faxkscxi91031pjkpccql6wspk4yrlnsbrrb5c0kamd5";
+  };
+  
+  buildInputs = [ ];
```
Comment (obadz) [inline pkgs/tools/admin/daemontools/default.nix]: Looks like this line doesn't do anything?

**Comment by obadz** [comment]
Builds fine here.

File: pkgs/tools/admin/daemontools/default.nix (hunk 1 above)
Comment (kevincox) [inline pkgs/tools/admin/daemontools/default.nix]: Good call. I'll remove that.

Comment (kevincox) [inline pkgs/tools/admin/daemontools/default.nix]: Fixed.

**Pull request merged**